### ISSUE TITLE
[A11y 8/25] Accessibility of the dropdown menu

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-dropdown-menu#30]
With Voiceover/Chrome: I'm not getting feedback that this is a dropdown menu. When I then open the menus, I can't use the screen reader to navigate through the items. I honestly wouldn't know that this dropdown was there if I was non-visual. 

On Chrome OS / Chromevox, this isn't as much of an issue. It tells me that ""group has popup"", instead of letting me know it's a dropdown or combobox. It tells me that the group has popup every time I try to navigate to the next item in the list though. This is really verbose. 

Still need to test experience on Windows.

**Comment by azakus** [comment]
Looks like keys now work to cycle through the items
@cdata, did something change in one of the dependencies?

**Comment by azakus** [comment]
combobox role looks pretty busted in VoiceOver, where the only examples I could find that actually work are ones where `<input>` has the combobox role: http://oaa-accessibility.org/examplep/combobox1/
Any wrapping breaks the "combobox" annoucement: http://oaa-accessibility.org/examplep/combobox4/

@cdata We may have to discuss some ugly DOM tricks / repeated subtress.

**Comment by lpalmaro** [comment]
Friendly ping -- any updates on making this accessible?

**Comment by azakus** [comment]
I played a lot with this, and the results were pretty grim. The only screenreader that supported our use of combobox correctly was NVDA. JAWS mostly worked, and both ChromeVox and VoiceOver failed spectacularly.

I think we need to give up on the combobox role, and instead try to make dropdown-menu just be a listbox that has a popup.

We need to hide the input somehow because it is just for decoration, but VoiceOver *really* likes to focus it for no particular reason.

**Comment by lpalmaro** [comment]
+Alice -- how does this approach sound to you?
